sync_diff_inspector/source/basic.go

```diff
@@ -50,6 +51,23 @@ func (s *BasicSource) GetRangeIterator(r *splitter.RangeInfo, analyzer TableAnal
 func (s *BasicSource) Close() {
 	s.dbConn.Close()
 }
+func (s *BasicSource) GetCountAndCrc32(ctx context.Context, tableRange *splitter.RangeInfo, countCh chan int64, checksumInfoCh chan *ChecksumInfo) {
```

Comment:
GetCrc32 can be rewriten by GetCountAndCrc32

I think we just keep one of them is OK.